src/main/webapp/dev/js/main/feedbackSubmissionsEdit.es6

```diff
@@ -963,10 +963,12 @@ $(document).ready(() => {
     if (typeof richTextEditorBuilder !== 'undefined') {
         $.each(textFields, (i, textField) => {
             const id = $(textField).attr('id');
+            const isSessionClosed = $(textField).data('is-session-closed');
```

Comment:
@amarlearning I know that, which is why I said just in case 😅. This is something I picked up from @whipermr5. (If you're interested you can see the discussion [here](https://github.com/TEAMMATES/teammates/pull/7715#issuecomment-316595294)). Even if we know that something can never be undefined, it never hurts to be prepared for the exception.